src/bitcoind.js

```diff
@@ -79,6 +79,7 @@ export async function bitcoindListUnspent({ url, auth, address, addresses }) {
     });
   } catch (e) {
     console.error("There was a problem:", e.message);
+    return e;
```

Comment:
maybe just `return` without actually returning the error. Probably better practice to actually return the error but in order to maintain the previous behavior?

Reply:
I can do that if you want, I don't feel strongly either way.